api-contracts/workflows/workflows.proto

```diff
@@ -44,6 +44,7 @@ message CreateWorkflowVersionOpts {
     optional StickyStrategy sticky = 12; // (optional) the sticky strategy for assigning steps to workers
     optional WorkflowKind kind = 13; // (optional) the kind of workflow
     optional int32 default_priority = 14; // (optional) the priority of the workflow
+    optional string event_filter_expression = 15; // (optional) the event expression for the workflow
```

Comment:
This should be a v1 workflow-only feature right?

Reply:
curious what the thinking is for adding to the v0 workflow contracts

Reply:
ah whoops I meant to delete this, put it in the wrong place originally and then moved it after - thanks!